src/wat-writer.cc

```diff
@@ -1743,6 +1743,9 @@ bool WatWriter::IsInlineExport(const Export& export_) {
     case ExternalKind::Tag:
       index = module.GetTagIndex(export_.var);
       break;
+
+    default:
+      WABT_UNREACHABLE;
```

Comment:
Doesn't adding this default case mean we don't get nice errors when we don't have full case coverage?  (e..g when we add a new extern kind?)

Reply:
Well, that is shame.  I'd rather not loose those warnings.   Kind of shame that gcc can't figure that out but clang can.

What happens if you add explicit `Fist` and `Last` cases here instead (with a comment explaining why)?

Reply:
Is this a bug in gcc?

Reply:
> What happens if you add explicit `Fist` and `Last` cases here instead (with a comment explaining why)?

GCC (and probably clang) complains about duplicate labels in the switch statement...

Reply:
If GCC can't give the error/warning fidelity we want here perhaps we should simply disable that particular warning under GCC?

Reply:
In the latest version, I kept the switches as-is and default-initialized the variable instead (in two places). It seems like it would be nice not to have to suppress "may be used uninitialized" warnings completely...?